package.json

```diff
@@ -2,12 +2,43 @@
   "name": "trunkio-plugins",
   "version": "1.0.0",
   "scripts": {
-    "trunk": "trunk"
+    "trunk": "trunk",
+    "build-test": "tsc",
+    "test": "jest --runInBand",
```

Comment:
You shouldn't need to build before running jest if you use `ts-jest`. Previously, I thought we were generating other files, so I didn't even think about it.